### DISCUSSION TITLE
Telegram bot for your portfolio 🤖

### DISCUSSION THREAD

**Discussion by MadeInPierre** [MadeInPierre/finalynx#125] [Show and tell]
Hey!

I just added a way to export your portfolio as a PNG file in Finalynx. This enabled me to write an external script (outside of this repo for now) to create a Finalynx Bot in Telegram : 

<img src="https://github.com/MadeInPierre/finalynx/assets/12670280/3731ff39-6938-43f6-937e-dda176db8373" width=200/>

Pretty fun :smile: If anyone else is also interested, should this telegram bot feature be integrated into Finalynx? If not, where should I share this telegram bot script?

**Comment by sebfar9172** [comment]
Hello
I am interrested

**Reply by MadeInPierre** [reply]
I added my telegram bot code along with setup instructions in `finalynx/examples/telegram_bot.py` :wink:
